yw_website/yw_website/apps/website/tests.py

```diff
@@ -121,3 +121,15 @@ def test_bad_workflow_update(self):
         response = self.client.post(route, data, format='json')
         self.assertEqual(response.status_code, 404,
                          msg="Workflow does not exist")
+
+    def test_no_repeating_tags(self):
+        data = copy.deepcopy(self.data)
+        data['tags'].append("tag1")
+        route = '/save/'
+
+        response = self.client.post(route, data, format='json')
+
+        route = '/save/{}/'.format(response.data['workflowId'])
+        response = self.client.post(route, data, format='json')
+        self.assertEqual(response.status_code, 200,
+                         msg="Error handling repeated tags")
```

Comment:
we know the error is with repeating tags based on the test name, it might be better to have `msg=response.data` so we can see the error message generated

Reply:
, we also probably want to validate that there is actually only one tag associated with that workflow, this just checks if there is some sort of error. The error could be anything.

Reply:
Will do